chapter_01/p03_urlify.py

```diff
@@ -21,6 +21,21 @@ def urlify_algo(string, length):
     return "".join(char_list[new_index:])
 
 
+
+def url_Ify(str1,length):
+    """ check each char and replace  spaces with %20  Time Complexity = o(n) """
```

Comment:
Appending to strings in python creates a new string every time. So complexity might actually be n^2